client/src/panels/internal_services/isc.js

```diff
@@ -81,27 +69,26 @@ export const declare_internal_services_panel = () => declare_panel({
       const {
         subject,
         graph_args: {
-  
           gov_fte_total,
           gov_isc_fte,
           total_fte,
           isc_fte,
-  
           series,
         },
       } = calculations;
-  
+
       const more_footnotes = [{
         text: GlossaryEntry.lookup("INT_SERVICES").definition,
       }].concat(footnotes);
```

Comment:
I'm not sure if we want to keep this hardcoded footnote instead of using the standard glossary tooltip that we usually use for glossary definitions. @Stephen-ONeil ?

Reply:
I agree, moving it to a glossary tooltip in the panel text would be better 👍

Reply:
👍👍 thanks for the good suggestions kate!